ComponentKitTestHelpers/CKTestRunLoopRunning.mm

```diff
@@ -25,9 +25,9 @@ BOOL CKRunRunLoopUntilBlockIsTrue(BOOL (^block)(void))
   CFTimeInterval timeoutDate = CACurrentMediaTime() + kTimeoutInterval;
   BOOL passed = NO;
   while (true) {
-    OSMemoryBarrier();
+    std::atomic_thread_fence(std::memory_order_seq_cst);
```

Comment:
This resolves a future deprecation warning. Likely not important now but I ran into it when running tests locally. Sequential consistency felt like the right memory order here but please let me know if my assumption is incorrect.